src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -209,17 +224,29 @@ protected int doWork() {
             baseQHisto.increment(i, baseQHistogramArray[i]);
         }
 
-        final WgsMetrics metrics = generateWgsMetrics();
-        metrics.GENOME_TERRITORY = (long) histo.getSumOfValues();
-        metrics.MEAN_COVERAGE = histo.getMean();
-        metrics.SD_COVERAGE = histo.getStandardDeviation();
-        metrics.MEDIAN_COVERAGE = histo.getMedian();
-        metrics.MAD_COVERAGE = histo.getMedianAbsoluteDeviation();
+        // Construct and write the outputs
+        final Histogram<Integer> baseQHetHisto = new Histogram<Integer>("value", "baseq_count");
+        for (int i=0; i<baseQHetSensHistogram.length; ++i) {
+            if(i>0 && i<17) {
```

Comment:
The `i > 0` part of this condition has no effect, since if `i==0` either branch will perform `increment(0, baseQHetSensHistogram[0])`.